cmd/generate/cmd_util.go

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) F5, Inc.
+ *
+ * This source code is licensed under the Apache License, Version 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/nginxinc/nginx-go-crossplane/internal/generator"
+)
+
+type filterFlag struct {
+	filter map[string]struct{}
+}
+
+func (reader *filterFlag) Set(value string) error {
+	if reader.filter == nil {
+		reader.filter = make(map[string]struct{})
+	}
+	reader.filter[value] = struct{}{}
+	return nil
+}
+
+func (reader *filterFlag) String() string {
+	return fmt.Sprintf("%v", *reader)
+}
+
+type overrideItem struct {
+	directive string
+	masks     []generator.Mask
+}
+
+func (item *overrideItem) UnmarshalText(text []byte) error {
+	rawOverride := string(text)
```

Comment:
Nit: The [`bytes` package](https://pkg.go.dev/bytes) may offer all of the functions from the `strings` package that you may be able to use to avoid the (very small) cost of casting `[]byte` to `string`